packages/search/lib/commands/ALIASDEL.ts

```diff
@@ -1,3 +1,5 @@
+export const FIRST_KEY_INDEX = 1;
```

Comment:
should be `2`?

Reply:
though about this for a while:

there could be a setup with each redis-cluster-node having an index with the same name. 

to create an alias on each node, the user can then control/select select the redis-cluster-node via hash tags (on the alias name) and manage aliases.


---


besides (im not sure), isn't redis cluster always using the first key after the command to calulate the hash slot internally? and might reply with MOVED

Reply:
both alias and index need to be same slot, else

```
127.0.0.1:6379> FT.ALIASADD test:{node-2} idx:{node-1}
(error) CROSSSLOT Keys in request don't hash to the same slot
```

changed FIRST_KEY_INDEX to 2 as seems more reasonable 👍